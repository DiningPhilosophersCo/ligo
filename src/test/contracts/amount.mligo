let check_ (p : unit) : int = if Tezos.amount = 100tz then 42 else 0
