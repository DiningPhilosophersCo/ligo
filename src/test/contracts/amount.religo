let check_ = (p : unit) : int =>
  if (Tezos.amount == 100tez) { 42; } else { 0; };
