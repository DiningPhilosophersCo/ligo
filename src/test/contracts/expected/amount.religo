let check_ = 
  (p: unit): int => 
    if(Tezos.amount
    == 100000000mutez) {
      42
    } else {
      0
    };
