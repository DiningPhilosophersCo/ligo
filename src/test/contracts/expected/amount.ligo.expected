function check (const p : unit) : int is
block {
  var result : int := 0;
  if amount = 100000000mutez
  then result := 42
  else result := 0
} with result
